src/main/resources/db/changelog/ddl/0018_EIP1-4481_alter_photo_location_arn_column_in_tables.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
+
+    <changeSet author="vishal.gupta@valtech.com" id="0018a_EIP1-4481_add_photo_location_arn_to_certificate_table - add photo_location_arn column" context="ddl">
+        <addColumn tableName="certificate">
+            <column name="photo_location_arn" type="varchar(1024)" afterColumn="status">
+                <constraints nullable="false"/>
+            </column>
+        </addColumn>
+
+        <rollback>
+            <dropColumn tableName="certificate" columnName="photo_location_arn"/>
+        </rollback>
+    </changeSet>
+
+
+    <changeSet author="vishal.gupta@valtech.com" id="0018b_EIP1-4481_certificate_table_data_migration - populate photo_location_arn column of certificate">
+        <sql>
+            <comment>Populating [photo_location_arn] of a certificate entity from any of the matching print_requests [photo_location_arn]</comment>
+            UPDATE certificate cert
+            INNER JOIN print_request pr on cert.id = pr.certificate_id
+            SET cert.photo_location_arn = pr.photo_location_arn
+            WHERE cert.photo_location_arn IS NULL
+        </sql>
+
+        <rollback/>
+    </changeSet>
+
+    <changeSet author="vishal.gupta@valtech.com" id="0018c_EIP1-4481_remove_photo_location_arn_column_in_print_request_table - Drop photo_location_arn column" context="ddl">
+        <dropColumn tableName="print_request" columnName="photo_location_arn"/>
```

Comment:
⚠️ Once the data is copied over to a new column, we remove the column from `print_request`